app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -148,6 +148,21 @@ <h2>Algorithm Result</h2>
                                     </a>
                                 </li>
                             {% endfor %}
+                            {% if input.file %}
+                                <li>
+                                    <a class="badge badge-primary"
+                                       href="{{ input.file.url }}" download="{{ input.file.name|name }}">
+                                        <i class="fa fa-download"></i>
+                                        Input {{ forloop.parentloop.counter }} {{ input.file.name|stem }}
+                                        ({{ input.file|suffix }})
+                                    </a>
+                                </li>
+                            {% endif %}
+                            {% if input.value %}
```

Comment:
Falsey values do not show up (eg, boolean is False here)

![image](https://user-images.githubusercontent.com/12661555/111641330-55269680-87fd-11eb-8fb7-890486e96ccb.png)

Reply:
I think that this is still not working